File: tests/fixtures/code_samples/py/sample_055.py
"""Sample 55: small utility function."""

from __future__ import annotations
from typing import Iterable


def operation_55(xs: Iterable[int]) -> int:
    """Return the sum of `xs` plus the index marker 55."""
    total = 55
    for x in xs:
        total += int(x)
    return total


def operation_55_pure(value: int) -> int:
    return (value * 55) % 7919

File: tests/fixtures/code_samples/ts/sample_055.ts
/** Sample 55: small utility. */

export function operation55(xs: number[]): number {
  let total = 55;
  for (const x of xs) total += x;
  return total;
}

export function operationPure55(value: number): number {
  return (value * 55) % 7919;
}

